### PR TITLE
AFL tweaks

### PR DESCRIPTION
**Pull request by dnoursi**
Fixing some bugs which broke AFL

### CODE REVIEW

File: engine-comparison/runner.sh [hunk 1]
```diff
@@ -30,6 +30,9 @@ if [[ $FUZZING_ENGINE == "afl" ]]; then
   if [[ !$(find seeds -type f) ]]; then
     echo "Input" > ./seeds/nil_seed
   fi
+  # TODO: edit core_pattern in Docker VM
+  # https://groups.google.com/forum/m/#!msg/afl-users/7arn66RyNfg/BsnOPViuCAAJ
+  export AFL_I_DONT_CARE_ABOUT_MISSING_CRASHES=1
```
Comment (morehouse) [inline engine-comparison/runner.sh:35]: Does this suppress crashes that we want to detect?

Comment (dnoursi) [reply engine-comparison/runner.sh:35]: I also wasn't sure, I discussed this with @kcc and it sounds fine for now because it works

Comment (morehouse) [reply engine-comparison/runner.sh:35]: So it can still detect all current benchmarks?

Comment (dnoursi) [reply engine-comparison/runner.sh:35]: I'm not sure, I found that it was necessary to begin fuzzing, but I haven't yet tested it to completion on all or even most benchmarks

**Review by morehouse — APPROVED** [review]

**Pull request merged**